idaes/core/util/misc.py

```diff
@@ -213,6 +215,94 @@ def copy_port_values(destination, source):
                 v[i].value = pyo.value(source.vars[k][i])
 
 
+def set_param_from_config(b, param, config=None, index=None):
+    """
+    Utility method to set parameter value from a config block. This allows for
+    converting units if required. This method directly sets the value of the
+    parameter.
+
+    Args:
+        b - block on which parameter and config block are defined
```

Comment:
I know this is just an argument for the method but `b` can be block to make it clear.

Reply:
What if `b` being passed is not an instance of `block`?

Reply:
If `b` is not a block, you need to read the documentation better. On the subject of `b` versus `block`, I think that should be part of the broader discussion @lbianchi-lbl started to standardize everything.

Reply:
> 
> 
> If `b` is not a block, you need to read the documentation better. On the subject of `b` versus `block`, I think that should be part of the broader discussion @lbianchi-lbl started to standardize everything.

Assuming someone constantly looks at the documentation. ;)